### PR TITLE
docs(zh-cn): Reviewed No. 10 - Instantiate a Transformers model (PyTorch)

### PR DESCRIPTION
**Pull request by PowerChina**
#390

### CODE REVIEW

**Comment by PowerChina** [comment]
@chenglu @lewtun @xianbaoqian Can you help to review and merge my first PR? Thank you for your help.

File: subtitles/zh-CN/10_instantiate-a-transformers-model-(pytorch).srt [hunk 1]
```diff
@@ -10,107 +10,107 @@
...
 11
 00:00:30,690 --> 00:00:33,810
-正如我们所见，当给定一个 BERT 检查点时
+正如我们所见，当给定一个 BERT 检查点时，
 As we can see, when given a BERT checkpoint
 
 12
 00:00:33,810 --> 00:00:38,043
-我们最终得到一个 BertModel，类似地，对于 GPT-2 或 BART。
+我们最终会得到一个 BertModel ，类似地，模型GPT-2 或 BART 也可以这么做。
```
Comment (chenglu) [inline subtitles/zh-CN/10_instantiate-a-transformers-model-(pytorch).srt:58]:
```suggestion
我们最终会得到一个 BertModel ，类似地，模型 GPT-2 或 BERT 也可以这么做。

```

always add a space between the CN and EN words.

File: subtitles/zh-CN/10_instantiate-a-transformers-model-(pytorch).srt [hunk 2]
```diff
@@ -10,107 +10,107 @@
...
 we end up with a BertModel and similarly, for GPT-2 or BART.
```
Comment (chenglu) [inline subtitles/zh-CN/10_instantiate-a-transformers-model-(pytorch).srt:59]:
```suggestion
we end up with a BertModel and similarly, for GPT-2 or BERT.

```

This is an issue of the original English subtitle, could you please open a PR for this?

**Review by chenglu — CHANGES_REQUESTED** [review]
Great work on the translation! I suggested a few format issues, please take a look and let me know if you have any questions!

File: subtitles/zh-CN/10_instantiate-a-transformers-model-(pytorch).srt (hunk 1 above)
Comment (PowerChina) [reply subtitles/zh-CN/10_instantiate-a-transformers-model-(pytorch).srt:58]: ok

**Review by lewtun — APPROVED** [review]
Great job @PowerChina and welcome to the Hugging Face community!

**Comment by PowerChina** [comment]
@lewtun thanks!

**Pull request merged**